src/shims/intrinsics.rs

```diff
@@ -654,6 +654,11 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
 
             "try" => return this.handle_try(args, dest, ret),
 
+            "breakpoint" => {
+                // normally this would raise a SIGTRAP, which aborts if no debugger is connected
```

Comment:
Ah, I noticed one more thing -- the arg count check is missing.
```suggestion
                let &[] = check_arg_count(args)?;
                // normally this would raise a SIGTRAP, which aborts if no debugger is connected
```